pyodide-build/pyodide_build/create_pypi_index.py

```diff
@@ -4,15 +4,15 @@
 from pyodide_lock.spec import PackageSpec
 
 
-def create_pypa_index(
+def create_pypi_index(
```

Comment:
I think this was meant to say "python package authority" = pypa not "python package index" = pypi. pypi is a specific package index. Maybe rename it to just `create_index` and call it `index` everywhere?

Reply:
Makes sense. Let's call it `package index` then.